### DISCUSSION TITLE
Schedule

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs424-program-reasoning#457] [Announcements]
Hi, here is the schedule for the rest of this semester:

* 12/4 (Wed): HW5 due
* 12/4 (Wed): No class
* 12/6 (Fri): All bonus activities due
* 12/7 (Sat): HW6 out
* 12/9 (Mon): Award session + Wrap up
* 12/11 (Wed): No class
* 12/16 (Mon): Final exam
* 12/18 (Wed): HW6 due

**Comment by JustinKim98** [comment]
I think (12/6) is Friday, not wednesday

**Reply by spearo2** [reply]
수정하였습니다. 감사합니다.

---

Fixed, Thank you
